Polyhedron/demo/Polyhedron/Plugins/Convex_decomposition/Nef_plugin.cpp

```diff
@@ -106,26 +111,36 @@ Polyhedron_demo_nef_plugin::on_actionToNef_triggered()
 
   Scene_polyhedron_item* item =
     qobject_cast<Scene_polyhedron_item*>(scene->item(index));
+  Scene_surface_mesh_item* sm_item =
+    qobject_cast<Scene_surface_mesh_item*>(scene->item(index));
+  if(!item && !sm_item)
+    return;
```

Comment:
There is an indentation issue of the whole block after this `return`, and that gives a warning:
```
[ 36%] Building CXX object demo/Polyhedron/Plugins/Convex_decomposition/CMakeFiles/nef_plugin.dir/Nef_plugin.cpp.o
/home/lrineau/Git/cgal/Polyhedron/demo/Polyhedron/Plugins/Convex_decomposition/Nef_plugin.cpp: In member function ‘void Polyhedron_demo_nef_plugin::on_actionToNef_triggered()’:
/home/lrineau/Git/cgal/Polyhedron/demo/Polyhedron/Plugins/Convex_decomposition/Nef_plugin.cpp:116:3: warning: this ‘if’ clause does not guard... [-Wmisleading-indentation]
   if(!item && !sm_item)
   ^~
/home/lrineau/Git/cgal/Polyhedron/demo/Polyhedron/Plugins/Convex_decomposition/Nef_plugin.cpp:119:5: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the ‘if’
```